Realm/RLMAccessor.mm

```diff
@@ -549,40 +549,40 @@ static void RLMSuperSet(RLMObjectBase *obj, NSString *propName, id val) {
     superSetter(obj, prop.setterSel, val);
 }
 
-// getter/setter for unmanaged
-static IMP RLMAccessorStandaloneGetter(RLMProperty *prop, RLMAccessorCode accessorCode) {
+// getter/setter for standalone
```

Comment:
@austinzheng's been working on moving to consistently refer to standalone objects as "unmanaged"